modules/steam/network.nix

```
@@ -0,0 +1,24 @@
+{lib, config, pkgs, ...}: let
+  cfg = config.jovian.steam;
+in with lib; {
+  options.jovian.steam.network = {
```

```suggestion
{lib, config, ...}:

let
  inherit (lib)
    /* add the necessary bits */
  ;
  cfg = config.jovian.steam;
in
{
  options.jovian.steam.network = {
```

Two things here:

1. the coding style should match the other expressions. (yeah, a style guide should be whipped-up for CONTRIBUTING.md after the docs update is in...)
2. A large-scoped `with` like that is undesirable, as it makes the origin of things harder to reason about.